workflow/envs/tabix.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
+  - htslib=1.10
```

Comment:
tabix in common container is tabix=1.11 . Will that match htslib=1.10?

Reply:
changed to `1.15.1`